src/expressions/operator-expr.md

```diff
@@ -385,7 +385,12 @@ reference types and `mut` or `const` in pointer types.
 | [Function pointer]    | Integer               | Function pointer to address cast |
 | Closure \*\*\*        | Function pointer      | Closure to function pointer cast |
 
-\* or `T` and `V` are compatible unsized types, e.g., both slices, both the same trait object.
+\* or `T` and `V` are unsized types with compatible metadata:
+* both slice metadata (`*[u16]` -> `*[u8]`, `*str` -> `*(u8, [u32])`).
+* both the same trait object metadata, modulo dropping auto traits (`*dyn Debug` -> `*(u16, dyn Debug)`, `*dyn Debug + Send` -> `*dyn Debug`).
+    * **Note**: *adding* auto traits is only allowed if the principal trait has the auto trait as a super trait
```

Comment:
```suggestion
    * **Note**: *adding* auto traits is only allowed if the principal trait has the auto trait as a super trait.
```

Reply:
Why is the dot needed here? The dot is currently after the parenthesis. Should it be moved?

Reply:
Got it, thanks; didn't see the continuation after the line break.